operations/iuf/workflows/upgrade_all_products.md

```diff
@@ -1,120 +1,393 @@
+
+1. Refer to the "Install and Upgrade Framework" section of each individual product's installation documentation to determine if any special actions need to be performed outside of IUF for the `deploy-product` or
+`post-install-service-check` stages.
+
+1. Invoke `iuf run` with `-r` to execute the [`deploy-product`](../stages/deploy_product.md) and [`post-install-service-check`](../stages/post_install_service_check.md) stages.
 
     (`ncn-m001#`) Execute the `deploy-product` and `post-install-service-check` stages.
 
     ```bash
-    iuf -a activity-all-products-upgrade -m "${ACTIVITY_DIR}" run --site-vars "${SITE_VARS_FILE}" --bootprep-config-dir "${ACTIVITY_DIR}/hpc-csm-software-recipe-23.05.0/vcs" -r deploy-product post-install-service-check
+    iuf -a "${ACTIVITY_NAME}" run -r deploy-product post-install-service-check
     ```
 
-    Once this step has completed:
+Once this step has completed:
+
+- New versions of product microservices have been deployed
+- Service checks have been run to verify product microservices are executing as expected
+- Per-stage product hooks have executed for the `deploy-product` and `post-install-service-check` stages
+
+### Manual configuration of the Slingshot fabric manager
+
+TBD
+
+## Managed nodes rollout
+
+This section updates the software running on managed compute and application (UAN, etc.) nodes.
+
+### Perform Slingshot switch firmware updates
+
+TBD
+
+### Execute the IUF `managed-nodes-rollout` stage
+
+**`NOTE`** Additional arguments are available to control the behavior of the `managed-nodes-rollout` stage. See the [`managed-nodes-rollout` stage documentation](../stages/managed_nodes_rollout.md) for details and adjust the
+examples below if necessary.
 
-    - new versions of product microservices have been deployed
-    - service checks have been run to verify product microservices are executing as expected
-    - per-stage product hooks have executed for the `deploy-product` and `post-install-service-check` stages
+1. Refer to the "Install and Upgrade Framework" section of each individual product's installation documentation to determine if any special actions need to be performed outside of IUF for the `managed-nodes-rollout` stage.
 
-1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage. The `-m`, `--site-vars`, and `--bootprep-config-dir` arguments are the same as the initial invocation of `iuf run` as all
-   steps in this example use the same IUF activity, media, and variables.
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on a single node to ensure the node reboots successfully with the desired image and CFS configuration. This node is
+referred to as the "canary node" in the remainder of this section. Use `--limit-managed-rollout` to target the canary node only and use `-mrs reboot` to reboot the canary node immediately.
```

Comment:
For canary nodes, do we want to recommend booting one per role type.
* Application 
* Compute
* Compute with Nvidia
* Compute with ROCM?

Reply:
Fixed in next update.